desktop-widgets/diveplanner.cpp

```diff
@@ -232,10 +253,34 @@ void DivePlannerWidget::heightChanged(const int height)
 	plannerModel->setSurfacePressure(pressure);
 }
 
-void DivePlannerWidget::salinityChanged(const double salinity)
+void DivePlannerWidget::waterTypeUpdateTexts()
+{
+	double density;
+	/* Do not set text in last/custom element */
+	for (int i = 0; i < ui.waterType->count()-1; i++) {
+		if (ui.waterType->itemData(i) != QVariant::Invalid) {
+			QString densityText = ui.waterType->itemText(i).split("(")[0].trimmed();
+			density = ui.waterType->itemData(i).toInt() / 10000.0;
+			densityText.append(QString(" (%1kg/ℓ)").arg(QLocale().toString(density, 'f', 2)));
```

Comment:
@Schwaneberg 
There was some discussion about using QLocale vs. using .arg with the `L` format specifier. Currently the decision is more in direction of the `L`.
So this should be please also changed to:
`densityText.append(QString(" (%L1%2)").arg(density, 0, 'f', 2).arg(tr("kg/ℓ")));`